source/manual/help-with-publishing-content.html.md

```diff
@@ -4,61 +4,53 @@ title: Help with publishing content
 
-### If emails haven't been sent for a published document
-
-If it looks as though emails haven't been sent out for a publication, there
-should be [an alert about unprocessed content changes and there is guidance on
-how to solve this](alerts/email-alert-api-app-healthcheck-not-ok.html#unprocessed-content-changes-content_changes).
-
-If there are no alerts, it would suggest that the content change never made it
-to the Email Alert API. In which case, you should [check that Email Alert
-Service is correctly consuming from RabbitMQ](alerts/rabbitmq-no-consumers-consuming.html).
-If all looks fine, it may be necessary to [manually create a `ContentChange` in
-Email Alert API](https://github.com/alphagov/email-alert-api/blob/1aee9703bf303d43ba4ecb5f6fd771b757d52daf/app/services/notification_handler_service.rb#L24-L43).
-
-We have an alert that checks that email alerts for Travel Advice and Medical
-Safety Alerts go out correctly. Although an alert won't be triggered for other
-kinds of documents, the [guidance will still apply](alerts/email-alerts-travel-medical.html).
-
-### [If scheduled publications haven't gone out][scheduled]
-
-If a scheduled publication hasn't gone out, you should start by following the
-instructions for [if documents aren't live after being published][live]. If it
-looks as though the content was never published from Whitehall, there is a
-Rake task available which will publish overdue documents.
-
-```bash
-$ bundle exec rake publishing:overdue:publish
-```
-
-[Run this job in production Jenkins ⚠️](https://deploy.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=whitehall&MACHINE_CLASS=whitehall_backend&RAKE_TASK=publishing:overdue:publish)
+- If it looks as though emails haven't been sent out for a publication, there
+  should be [an alert about unprocessed content changes and there is guidance on
+  how to solve this](alerts/email-alert-api-app-healthcheck-not-ok.html#unprocessed-content-changes-content_changes).
+
+  If there are no alerts, it would suggest that the content change never made it
+  to the Email Alert API. In which case, you should [check that Email Alert
+  Service is correctly consuming from RabbitMQ](alerts/rabbitmq-no-consumers-consuming.html).
+  If all looks fine, it may be necessary to [manually create a `ContentChange` in
+  Email Alert API](https://github.com/alphagov/email-alert-api/blob/1aee9703bf303d43ba4ecb5f6fd771b757d52daf/app/services/notification_handler_service.rb#L24-L43).
+
+  We have an alert that checks that email alerts for Travel Advice and Medical
+  Safety Alerts go out correctly. Although an alert won't be triggered for other
+  kinds of documents, the [guidance will still apply](alerts/email-alerts-travel-medical.html).
+
+- If [a scheduled publication hasn't gone live][scheduled-publishing],
+  you should start by following the instructions for [if documents
+  aren't live after being published][live]. If it looks as though the
```

Comment:
I'd suggest this is reworded:
```md
- If [a scheduled publication hasn't gone live][scheduled-publishing],
  you should start by following the instructions: [if documents
  aren't live after being published][live].
```

Reply:
You mean as a separate paragraph to the next sentence? OK.

Reply:
Wait... sorry, I really can't see the problem here!

Reply:
I think that "start by following the instructions for if documents aren't live after being published" doesn't flow very well. Accepting that the format may make it clear that "if documents" starts the description of the document being linked to, I just think the using a punctuation point like ":" would make that break more obvious. So the text reads:

> start by following the instructions: if documents aren't live after being published

Reply:
I fixed this up. Thanks!